sam-persistence/api-services.tf

```diff
@@ -0,0 +1,17 @@
+module "enable-services" {
+  source      = "github.com/broadinstitute/terraform-shared.git//terraform-modules/api-services?ref=services-0.3.0-tf-0.12"
+  enable_flag = "1"
+  providers = {
+    google.target = google.target
+  }
+  project = var.google_project
+  services = [
+    "cloudkms.googleapis.com",
+    "datastore.googleapis.com",
+    "pubsub.googleapis.com",
+    "sql-component.googleapis.com",
+    "sqladmin.googleapis.com",
+    "storage-api.googleapis.com",
+    "storage-component.googleapis.com",
+  ]
```

Comment:
Double checking, does Sam persistence need all these APIs? I'm not familiar with the persistence layer or where it stores stuff, so the answer could easily be "yes".

Reply:
Yeah, there's a *lot* of stuff that comes along with Sam. Adding some comments about where some of these requirements come from.